From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Paul Gofman <pgofman@codeweavers.com>
Date: Mon, 25 Mar 2024 13:36:21 -0600
Subject: [PATCH] kernelbase: HACK: Force CEF swiftshader for Snares of Ruin 2.

CW-Bug-Id: #23613
Valve-commit: 51ba676f7aa2e4e6d74be67a77840d4d1c203162
Picked: clean
Class: valve
Evidence: docs/evidence/2026-09-28-wine-proton-rebase.md
Offered-upstream: no
---
 dlls/kernelbase/process.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dlls/kernelbase/process.c b/dlls/kernelbase/process.c
index 5c27b4e..7ca7cb7 100644
--- a/dlls/kernelbase/process.c
+++ b/dlls/kernelbase/process.c
@@ -514,6 +514,7 @@ static const WCHAR *hack_append_command_line( const WCHAR *cmd )
         {L"Paradox Launcher.exe", L" --use-angle=gl"},
         {L"Montaro\\nw.exe", L" --use-gl=swiftshader"},
         {L"Aisling and the Tavern of Elves\\nw.exe", L" --use-gl=swiftshader"},
+        {L"Snares of Ruin 2\\SoR2.exe", L" --use-gl=swiftshader"},
         {L"OlympiaRising.exe", L" --use-gl=swiftshader"},
         {L"nw.exe.exe", L" --use-angle=d3d9"},
         {L"DC Universe Online\\LaunchPad.exe", L" --use-gl=swiftshader"},
